modules/gitops-argocd-cli-logging-in-to-argocd-server.adoc

```diff
@@ -0,0 +1,76 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/logging-in-to-argocd-server.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-logging-in-to-argocd-server{context}"]
+= Logging in to the Argo CD server
+
+After you install and configure the {gitops-shortname} `argocd` CLI, to execute commands in the default mode, you must log in to the Argo CD server.
```

Comment:
This sentence feels awkward. Consider rearranging it like so:

"After you install and configure the {gitops-shortname} `argocd` CLI, you must log in to the Argo CD server to execute commands in the default mode."

Alternatively, consider this:

"You must log in to the Argo CD server to execute commands in the default mode after installing and configuring the {gitops-shortname} `argocd` CLI."